syntax = "proto3";

import "region/v1/region.proto";

package gcv.services;


service ChromosomeRegion {
  rpc Get (ChromosomeRegionGetRequest) returns (ChromosomeRegionGetReply) {}
}


message ChromosomeRegionGetRequest {
  string chromosome = 1;
  uint32 start = 2;
  uint32 stop = 3;
}

message ChromosomeRegionGetReply {
  gcv.region.v1.Region region = 1;
}
